src/workerd/io/trace.c++

```diff
@@ -251,6 +253,50 @@ kj::String KJ_STRINGIFY(const InvocationSpanContext& context) {
   return kj::str(context.getTraceId(), "-", context.getInvocationId(), "-", context.getSpanId());
 }
 
+kj::String KJ_STRINGIFY(const tracing::TailEvent::Event& event) {
+  KJ_SWITCH_ONEOF(event) {
+    KJ_CASE_ONEOF(onset, tracing::Onset) {
+      return kj::str("Onset");
```

Comment:
Nit: Shouldn't we also stringify onset here? For solving the issue at hand this does not seem necessary (we've only seen SpanOpen/SpanClose cause trouble so far) but there should be at least a comment here in that case.